app/components/post/post.js

```diff
@@ -177,6 +181,10 @@ export default class Post extends React.Component {
         return attachments;
     }
 
+    viewUserProfile = () => {
+        this.props.actions.goToUserProfile(this.props.user.id);
```

Comment:
I think it would make more sense to mark the user prop as required since it's also used for the profile picture on the post.